jwst/background/tests/test_background_wfss.py

```diff
@@ -216,6 +216,9 @@ def shared_tests(sci, mask, original_data_mean):
     tol = 0.01 * np.nanstd(sci)
     assert np.isclose(np.nanmean(sci), original_data_mean, atol=tol)
 
+    # Verify that the mask extension in the datamodel is populated
+    assert dm.mask.all() == mask.all()
```

Comment:
shouldn't this be something like `np.assert_allclose(dm.mask, mask)`?

https://numpy.org/doc/2.1/reference/generated/numpy.all.html is collapsing each of these into a single boolean `True` or `False`